templates/download/risc-v/partner-built.html

```diff
@@ -0,0 +1,108 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Download Partner-built Ubuntu for RISC-V Platforms{% endblock %}
+
+{% block meta_description %}
+  Get partner-built Ubuntu images from companies who host their own Ubuntu images and  who have followed the RISC-V Image Cookbook.
+  All images hosted here do not have Canonical support and maintenance and should be for development/preview purposes only.
+{% endblock meta_description %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/11AUwjePeOGvT6H8xM4QYixzmUTRpG_uKPa_pSzR5ucs/
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% from "_macros/vf_hero.jinja" import vf_hero %}
+
+{% block content %}
+
+  {% call(slot) vf_hero(
+      title_text='Download Partner-built Ubuntu for <br class="u-hide--medium" /> RISC-V platforms',
+      layout='25/75'
+    ) -%}
+
+    {%- if slot == 'signpost_image' -%}
+      {{ image(url="https://assets.ubuntu.com/v1/4d5b8d51-riscv-color.svg",
+              alt="RISC-V",
+              width="200",
+              height="32",
+              hi_def=True,
+              loading="auto",
+              attrs={"style": "padding-top: 1rem"},) | safe
+      }}
+    {%- endif -%}
+
+    {%- if slot == 'description' -%}
+      <p>
+        These Ubuntu images are built and hosted by Canonical’s partners, using Canonical’s tools and standard build processes. 
+        They are provided as developer previews only, are not production‑ready ,and do not include Canonical’s security updates or support. 
+        If you’re building a commercial product and need support, please get in touch with Canonical.
+      </p>
+      <p>
+        To run Ubuntu on your RISC‑V board, follow the Partner’s instructions for programming your platform.
+      </p>
+    {%- endif -%}
+
+    {%- if slot == 'cta' -%}
+      <a href="/download/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
```

Comment:
There is supposed to be another CTA next to this button. Can you get the stakeholder's confirmation that it's not needed in the copy doc?